cost/azure/idle_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## v3.0
+
+- applying data normalization updates for spend recommendations api. this change breaks current iterations expecting specific output types being pushed.
+- fixed issue with potential nil pointer type error issue
+- updated savings field to round to 3rd decimal
+- Normalizing fields for recommendations:
+  - updated 'tags' field to slice of string values
+  - Renamed 'id' to 'resourceID'
+  - Added 'AccountID' field with subscriptionID value
+  - Added 'AccountName' field with subscriptionName value
+  - Changed 'savings' to be a plain number
+  - Added 'savings_currency' to record the currency symbol.
```

Comment:
Just noticed this field is different -- maybe we should switch it to use camel-case?
```suggestion
  - Added 'savingsCurrency' to record the currency symbol.
```

Reply:
Id prefer that, I was mirroring the AWS ones but I was looking to probably get those changed as well if you have no issues with it.